cpp/src/barretenberg/honk/sumcheck/sumcheck.test.cpp

```diff
@@ -196,19 +140,19 @@ TEST(Sumcheck, PolynomialNormalization)
                                                        lagrange_first,
                                                        lagrange_last);
 
-    auto transcript = produce_mocked_transcript(multivariate_d, num_public_inputs);
+    auto transcript = ProverTranscript<FF>::init_empty();
 
     auto sumcheck = Sumcheck<FF,
-                             Transcript,
+                             ProverTranscript<FF>,
                              ArithmeticRelation,
                              GrandProductComputationRelation,
                              GrandProductInitializationRelation>(multivariate_n, transcript);
 
-    sumcheck.execute_prover(full_polynomials);
+    auto [multivariate_query, evaluations] = sumcheck.execute_prover(full_polynomials, {});
```

Comment:
"query"?

Reply:
Yeah I've started to use `query` and `evaluation` for respectively, the point at which a polynomial is evaluated and the result of that evaluation. I'm open to other terminology I mostly just want to develop a clear convention.